### PR TITLE
zh:/news/security/istio-security-2022-003/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by yanrongshi** [comment]
@ericvn hey,brother,what is "did not find expected key" ? what shou i do in the next,thanks

**Review by ericvn — COMMENTED** [review]
I believe it's picking on the extra comma after on this line.

File: content/zh/news/security/istio-security-2022-003/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: ISTIO-SECURITY-2022-003
+subtitle: 安全公告
+description: Multiple CVEs related to istiod Denial of Service and Envoy.
+cves: [CVE-2022-23635, CVE-2021-43824, CVE-2021-43825, CVE-2021-43826, CVE-2022-21654, CVE-2022-21655, CVE-2022-23606]
+cvss: "7.5",
```
[the remark this answers is not in the document]
Comment (yanrongshi) [reply content/zh/news/security/istio-security-2022-003/index.md]: fine，thanks

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**